packages/contentful--app-scripts/lib/utils.js

```diff
@@ -13,6 +14,22 @@ const throwValidationException = (subject, message, details) => {
   throw new TypeError(message);
 };
 
+const validateIpAddress = (ipAddress) => {
```

Comment:
can we add some unit tests for these? It's a big regex and brittle. Also the name can be `isValidIpAddress` to hint that we're returning boolean

Reply:
Dont forget IPv6

Reply:
Isn't there a library for that? 😶 I wouldn't reinvent the wheel for such a big regex.

Reply:
I will look for the library.

Reply:
I searched for library, but unfortunately I couldn't find all the things which I need in one library